include/openssl/ec.h

```diff
@@ -460,6 +460,21 @@ EC_GROUP *EC_GROUP_new_curve_GF2m(const BIGNUM *p, const BIGNUM *a,
 EC_GROUP *EC_GROUP_new_from_params(const OSSL_PARAM params[],
                                    OSSL_LIB_CTX *libctx, const char *propq);
 
+/**
+ * Creates an OSSL_PARAM array with the parameters describing the given EC_GROUP.
+ * The resulting parameters may contain an explicit or a named curve depending
+ * on the EC_GROUP.
+ *  \param  group   pointer to the EC_GROUP object
+ *  \param  libctx The associated library context or NULL for the default
+ *                 context
+ *  \param  propq  A property query string
+ *  \param  bnctx  BN_CTX object (optional)
+ *  \return newly created OSSL_PARAM array with parameters with the parameters
+ *          describing the given EC_GROUP or NULL if an error occurred
+ */
```

Comment:
You can drop this comment once you add the proper documentation into the .pod file.

Reply:
I've kept it for now to be consistent with the other function in the same file. I can also remove it if you think it is redundant information.